### ISSUE TITLE
os x (and windows) compatibility for deploy.sh

### ISSUE THREAD

**Issue by sheppard** [wq/wq-django-template#20]
See wq/vera#4.

This might mean moving more functionality to Python for easier cross-platform compatibility.

**Comment by olivierdalang** [comment]
What about providing a basic Docker setup ?

It's very common nowadays and has the advantage that you can even adapt the setup for deployment.

If you like the idea I may have some time soon to contribute that.

**Comment by sheppard** [comment]
Yes, a Docker setup would be beneficial.  Would it make more sense to put it here or in the [wq.start repository](https://github.com/wq/wq.start)?.

**Comment by olivierdalang** [comment]
I'd put it here as it also depends on the code repository structure.
